app/models/application_dates.rb

```diff
@@ -12,7 +12,7 @@ def respond_by
   end
 
   def edit_by
-    7.business_days.after(submitted_at)
+    5.business_days.after(submitted_at)
```

Comment:
Given that specs are entering our world via the spreadsheet and cucumber, does this fix invite us to pull the cucumber spec in? (For subsequent PR?)